wp-content/plugins/ig-revisions/plugin.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Plugin Name: Integreat Page Revisions
+ * Description: Set a specific page revision for delivery via API
+ * Version: 1.0
+ * Author: Sven Seeberg
+ * Author URI: https://github.com/Integreat
+ * License: MIT
+ * Text Domain: ig-revisions
+ */
+
+function ig_revisions_metabox( $post ) {
+	$children = wp_get_post_revisions( $post_id );
+	$revision_id = get_post_meta( $post->ID, 'ig_revision_id', true );
+	$options = "<option value='-1'" . (!$revision_id ? " selected" : "") . ">None</option>";
+	
+	foreach($children as $child) {
+		$options .= "<option value='" . $child->ID . "'" . ($child->ID==$revision_ids ? " selected" : "" ) . ">" . $child->post_date . "</option>";
```

Comment:
`$revision_ids` should be `$revision_id` here, otherwise the selected revision won't be shown in the dropdown menu.